api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/UsMonthlySummaryQueries.java

```diff
@@ -0,0 +1,291 @@
+package org.openmrs.module.eptsreports.reporting.library.queries;
```

Comment:
This file takes in the only queries directly as from old report without passing those encounters at run time as we have done before?

Reply:
Those queries looks complex though, but is it possible to pass parameters in the cohort definition or at runtime or access the configurable properties we already set up in the config?

Reply:
sure, i want to do exactly that but didn't want to spend such time until am sure these queries are what we are going to retain, with TX TB, we will have to remove some of those i did for queries that ended up not being used after we did the user story changes